cafebabel/static/js/profile-contribution-switch.js

```diff
@@ -0,0 +1,47 @@
+const translationSwitcher = document.querySelector('.translation-switcher')
```

Comment:
I wonder if we really need this script.
The idea is to fade in and out the boxes, right?
I guess [this could do the job too](https://www.cssscript.com/creating-a-stylish-tabs-control-using-pure-css-css3/) in pure CSS, or be an inspiration.

Reply:
ok

Reply:
I don't like it but I did it. Since I couldn't get with opacity transition, I played with display only 👎